db/migrate/20231106151705_default_remixed_project_origin.rb

```diff
@@ -0,0 +1,20 @@
+class DefaultRemixedProjectOrigin < ActiveRecord::Migration[7.0]
+  def up
+    if Rails.env.development?
+      remix_origin = 'http://localhost:3010'
+    elsif Rails.env.test?
+      remix_origin = 'staging-editor.raspberrypi.org'
+    elsif Rails.env.production?
+      remix_origin = 'editor.raspberrypi.org'
+    end
+    Project.find_each do |project|
+      project.update_attribute(:remix_origin, remix_origin) unless project.remixed_from_id.nil?
+    end
```

Comment:
Depending on how many projects there are in production this might be a non-issue, but if there's quite a lot we could optimise this into a single query, perhaps something like (not tested this!):

```
Project.where.not(remixed_from_id: nil).update(remix_origin: remix_origin)
```

Reply:
Okay, I will give this a try 👀

Reply:
11314 rows in projects on production for what it's worth, so probably worth trying to do it in one query, but not disastrous if we can't I think.

Reply:
Worked a treat, thanks Chris 👍

Reply:
👍